notebooks/_01-ProblemStatement.md

```diff
@@ -19,4 +19,4 @@ jupyter:
 ## What is the problem motivating this project?
 
 
-This project attempts to bridge the gap between causal inference modeling practices and the field of demand modeling (discrete choice modeling) in transportation by demonstrating an approach to think causally about the modeling problem.
+This project will attempt to bridge the gap between causal inference modeling practices and the field of demand modeling (discrete choice modeling) in transportation by demonstrating an approach to think causally about the modeling problem.
```

Comment:
The prompt is to note what problems the project aims to solve, not what we will do to solve the problem.

Try to frame the response in terms of "___ is wrong" or "___ needs to be improved."

Reply:
@timothyb0912 @hassanobeid1994  I am thinking something along the lines of: 

The existing disconnect between the fields of travel demand modeling and causal inference make it hard for cross-fertilization between the two fields to take place. This project is motivated by the current lack of use of methods and findings from causal inference in travel demand modeling. 

Thoughts?